item-spec/item-spec.md

```diff
@@ -215,6 +221,105 @@ Deprecation notice: GeoTiff previously used the media type `image/vnd.stac.geoti
 Cloud Optimized GeoTiffs used `image/vnd.stac.geotiff; profile=cloud-optimized`.
 Both can still appear in old catalogues, but are deprecated and should be replaced.
 
+#### Additional Fields for Assets
+
+As detailed above, Items contain properties, which are the main source of metadata for searching across Items. Many content extensions can add further property fields as well. Any property that can be specified for an Item can also be specified for a specific asset. This can be used to override a property defined in the Item, or to specify fields for which there is no single value for all assets.
+
+**It is important to note that the STAC API does not faciliate searching across Asset properties in this way, and this should be used sparingly.** It is primarily used to define properties at the Asset level that may be used during use of the data instead of for searching. 
+
+For example, `gsd` defined for an Item represents the best Ground Sample Distance (resolution) for the data within the Item. However, some assets may be lower resolution and thus have a higher `gsd`. The `eo:bands` field from the EO extension defines an array of spectral bands. However, it may be useful instead to specify the bands that are used in a particular asset.
+
+Here is a partial Sentinel-2 Item:
+
```

Comment:
I'd say move this to be an actual example? I think we've moved away from including longer json examples directly in the spec - I just cleaned up catalog in this way. Just make an example, and then link to it and talk about it.